laze-project.yml

```diff
@@ -302,6 +302,7 @@ contexts:
     parent: esp
     selects:
       - xtensa
+      - something-inside-the-esp32s3-blobs-that-contains-a-c-abort
```

Comment:
That might be done prettier in laze, and if anyone has any insight as to whether a non-WiFi build of ESP32S2 would also have that in, that might also help.

Reply:
There's https://github.com/kaspar030/laze/issues/638#issue-2854170723 to track that in laze; until I get around to patching that, I think we'll need this workaround (but probably with a better name).